include/swift/AST/Builtins.def

```diff
@@ -880,6 +880,11 @@ BUILTIN_MISC_OPERATION_WITH_SILGEN(BuildDefaultActorExecutorRef,
 BUILTIN_MISC_OPERATION_WITH_SILGEN(BuildMainActorExecutorRef,
                                    "buildMainActorExecutorRef", "n", Special)
 
+/// isOnStack : (Builtin.NativeObject) -> Builtin.Int1
+///
+/// Returns true if the optimizer successfully moved this object from the stack onto the heap.
```

Comment:
Is this comment backwards?

Reply:
I think so but you'll have to ask @gottesmm since it's from his PR that I merged into this one